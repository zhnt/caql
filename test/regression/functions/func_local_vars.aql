function counter() {
    let count = 0
    count = count + 1
    print("Count: " + string(count))
}

function first_caller() {
    print("First caller:")
    counter()
    counter()
}

function second_caller() {
    print("Second caller:")
    counter()
}

first_caller()
second_caller()


