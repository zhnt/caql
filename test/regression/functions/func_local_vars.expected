First caller:
Count: 1
Count: 1
Second caller:
Count: 1

